pkg/config/app/kuma-cp/config.go

```diff
@@ -150,6 +151,8 @@ type Config struct {
 	Proxy xds.Proxy `json:"proxy"`
 	// Intercommunication CP configuration
 	InterCp intercp.InterCpConfig `json:"interCp"`
+	// Tracing
+	Tracing *tracing.Config `json:"tracing"`
```

Comment:
nit: why pointer

Reply:
yeah I can check if otel.endpoint is empty instead